File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.java
class Solution {
    public int reverseDegree(String s) {
        int n=s.length();
        long result=0;
        for(int i=0; i<n; i++){
            int temp=(i+1)*(26-(s.charAt(i)-'a'));
            result+=temp;
        }
        return (int)result;
    }
}

File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.cpp
class Solution {
public:
    int reverseDegree(string s) {
        int n=s.size();
        long long result=0;
        for(int i=0; i<n; i++){
            int temp=(i+1)*(26-(s[i]-'a'));
            result+=temp;
        }
        return result;
    }
};